components/support/migration/build.gradle

```diff
@@ -73,3 +78,6 @@ dependencies {
 
 apply from: '../../../publish.gradle'
 ext.configurePublish(config.componentsGroupId, archivesBaseName, project.ext.description)
+
+ext.gleanGenerateMarkdownDocs = true
+apply from: '../../service/glean/scripts/sdk_generator.gradle'
```

Comment:
Please note that the SDK generator is [deprecated since A-C 27](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md#2700). You need to use [this](https://github.com/mozilla-mobile/android-components/blob/cabd445aae1a995c7154c343301ce600788be85d/components/support/sync-telemetry/build.gradle#L11-L13) and [this](https://github.com/mozilla-mobile/android-components/blob/cabd445aae1a995c7154c343301ce600788be85d/components/support/sync-telemetry/build.gradle#L62-L63)